cg/meta/upload/scout/uploadscoutapi.py

```diff
@@ -455,3 +378,72 @@ def get_config_builder(self, analysis, hk_version) -> ScoutConfigBuilder:
         }
 
         return config_builders[analysis.pipeline]
+
+    def create_rna_dna_mapper(self, rna_case: models.Family) -> Dict[str, Dict[str, list]]:
+        """Build nested relationship dictionary for DNA samples and their DNA cases based on each RNA sample
+        subject_id in a case. This allows for upload of all RNA samples in an RNA case to all matching DNA samples in
+        multiple DNA cases. Example dict row: {rna_sample_id : {dna_sample_id : [dna_case_1, dna_case_2]}}. For a
+        trio of RNA samples matching a trio of DNA samples, it would have three of these rows.
```

Comment:
I think this is enough
```suggestion
        """Returns a nested dictionary for mapping an RNA sample to a DNA sample and its DNA cases based on subect_id. Example dictionary {rna_sample_id : {dna_sample_id : [dna_case1_id, dna_case2_id]}}.
```

Reply:
Thanks, added to next commit.